src/index.html

```diff
@@ -1,13 +1,78 @@
 <!doctype html>
```

Comment:
my advice to use UPPERCASE for !DOCTYPE
```suggestion
<!doctype html>
```